release/_package.sh

```diff
@@ -102,6 +102,11 @@ function package_boms {
 }
 
 function package_release {
+	if [ "${LIFERAY_RELEASE_PRODUCT_NAME}" == "portal" ]
+    then
+        rm -fr "${_BUNDLES_DIR}/routes"
+    fi
+	
```

Comment:
@pgydev, based on the discussion https://liferay.slack.com/archives/C01FP01V5L0/p1717444981094919, shouldn't we remove `/routes/default/dxp` only?